changelog/865.feature.rst

```diff
@@ -0,0 +1,26 @@
+.. Added aliases (with trailing underscores) for parameters in formulary::
```

Comment:
@StanczakDominik Can you look at this? Not sure how well this will behave with `towncrier`.

Reply:
I looked into how `towncrier` would render this and it does not come out nicely.  Formatting the change log like the following would generated a lot more cleanly...

```restructuredtext
Added aliases (with trailing underscores) for parameters in the formulary:

    * `plasmapy.formulary.dimensionless.Reynolds_number` -> `~plasmapy.formulary.dimensionless.Re_`
    * `plasmapy.formulary.dimensionless.Mag_Reynolds`    -> `~plasmapy.formulary.dimensionless.Rm_`
    * ...
```

Reply:
On it. I've never written a change log before. Sorry the format isn't working nicely.

Reply:
No worries.  `towncrier` is still a bit of a black box to myself as well.  If it helps, I think of the change log as a bullet point in a reST list.